java/code/src/com/suse/manager/webui/controllers/virtualization/VirtualGuestsController.java

```diff
@@ -132,6 +134,8 @@ public void initRoutes(JadeTemplateEngine jade) {
                 withUserPreferences(withCsrfToken(withDocsLocale(withUser(this::console)))), jade);
         get("/manager/api/systems/details/virtualization/guests/:sid/data",
                 withUser(this::data));
+        post("/manager/api/systems/details/virtualization/guests/:sid/refresh",
```

Comment:
you can keep it all in one line, we can go up to 140 characters IIRC :wink:

Reply:
IIRC it's 120, but that line fits both: fixed